### ISSUE TITLE
Chrome - Big data: when trying to export a large set of data the export is not succesful and there is no error

### ISSUE THREAD

**Issue by KacperMadej** [highcharts/export-csv#107]
On Chrome - When trying to export a large set of data the export fails and the file cannot be downloaded.

Demo: http://jsfiddle.net/cqjvD/1260/
Steps: 
1. download CSV
2. file is not download and a browser informs about network error.

Is there a known limit for the exported data? 
150k data points seems to be too big - the fail occurs.

Could there be at least some kind of info about the fail?

Browsers:
Chrome - the affected browser,
IE11 and Edge - hangs for a while but handles it,
Firefox - runs smoothly even with 1 million points.

**Comment by TorsteinHonsi** [comment]
I got this error from the export server:
![skjermbilde 2016-11-07 kl 09 58 26](https://cloud.githubusercontent.com/assets/227836/20051727/ce9c7346-a4d0-11e6-9c78-e2f9589c9a4c.png)

**Comment by TorsteinHonsi** [comment]
#### Workaround
Use the boost module, works like a breeze: http://jsfiddle.net/highcharts/cqjvD/1261/

**Comment by pan-henryk** [comment]
Interesting. I got a bug similar to Kacper, but after investigation boost module turned out to be the culprit.
Whenever your data exceeds boostThreshold, csv export returns empty data set. (It happens both in my project and your jsfiddle: http://jsfiddle.net/highcharts/cqjvD/1261/
(internally getDataRows seems to return only two rows then).

**Comment by LaniMoo** [comment]
Hi there,

I ran into this problem too except when trying export large files as xls. I came across this solution and it worked for me: https://stackoverflow.com/questions/39374157/angular-save-file-as-csv-result-in-failed-network-error-only-on-chrome

Any chance we can get this fix in?

**Comment by nikolas** [comment]
Yeah - I'm still seeing this problem with `export-data` from HighCharts 6.1.0 with Chromium 62.

**Comment by KacperMadej** [comment]
Hi @nikolas 

The repository is depreciated.

> Given the popularity of this plugin, it has been taken in as a Highcharts module since v5.0.11 (2017-05-04), and development continues in the official Highcharts repo. This means it can be loaded from code.highcharts.com and is available with the Highcharts npm package. Issues should now be reported in the Highcharts repo.

https://github.com/highcharts/highcharts/issues
